remote-trait-object/src/service/serde_support.rs

```diff
@@ -81,29 +82,37 @@ pub(crate) mod port_thread_local {
     }
 }
 
-impl<P: ?Sized + Service, T: ToDispatcher<P>> Serialize for ServicePointer<T, P> {
+impl<T: ?Sized + Service> Serialize for ServiceRef<T> {
     fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
     where
         S: Serializer, {
-        let service = self.take();
-        let handle = port_thread_local::get_port().upgrade().unwrap().register(T::to_dispatcher(service));
-        handle.serialize(serializer)
+        let error = "You must not de/serialize ServiceRef by yourself.";
```

Comment:
When the user implement ServiceRef on their own?
If the user should implement ServiceRef when they want to serialize `Import` service ref?